build.gradle

```diff
@@ -118,6 +118,9 @@ subprojects {
       ':distribution:tools:plugin-cli',
       ':qa:os',
       ':qa:wildfly',
+      ':qa:die-with-dignity',
+      ':qa:logging-config',
+      ':qa:evil-tests',
```

Comment:
I've changed this to an opt-out list in `gradle/formatting.gradle`, so you should remove these project paths from the list in that file.